src/main/java/mate/academy/springbootwebgreqit/dto/category/UpdateCategoryRequestDto.java

```diff
@@ -1,10 +1,13 @@
 package mate.academy.springbootwebgreqit.dto.category;
 
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
 import lombok.Data;
 
 @Data
 public class UpdateCategoryRequestDto {
     private Long id;
```

Comment:
Olena meant to remove the id field entirely because you passed it as a PathVariable.

Reply:
but i use category id in CategoryService
![image](https://github.com/user-attachments/assets/f90b936c-d7c7-4736-b068-e93676fa2c46)

Reply:
You should provide not only UpdateDto, but also Long id that you get from path variable

Reply:
`Long id` still here